src/main/java/com/epam/learning/backendservices/kafka/AvroProducer.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.learning.backendservices.kafka;
+
+import org.apache.kafka.clients.producer.*;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class AvroProducer {
+
+    private static final Logger logger = Logger.getLogger(AvroProducer.class.getName());
+
+    public static void main(String[] args) {
+
+        Properties properties = new Properties();
+        properties.setProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
```

Comment:
Optional: add docker-compose.yml file. So I can run it

Reply:
Done